### PR TITLE
pi-gen automation - downloadable images

### PR DESCRIPTION
**Pull request by JarbasAl**
Learn more at [RPi-Distro/pi-gen](https://github.com/RPi-Distro/pi-gen)

### CODE REVIEW

File: stage-picroft/00-build/01-run.sh [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+echo "tmpfs /ramdisk tmpfs rw,nodev,nosuid,size=20M 0 0" >> "${ROOTFS_DIR}/etc/fstab"
+
+# console auto login
+install -v -m 0644 files/etc/systemd/system/getty@tty1.service.d/autologin.conf "${ROOTFS_DIR}/etc/systemd/system/getty@tty1.service.d/autologin.conf"
+
+# install picroft files
+install -v -d -m 0755 "${ROOTFS_DIR}/etc/mycroft"
+install -v -m 0644 files/etc/mycroft/mycroft.conf "${ROOTFS_DIR}/etc/mycroft/mycroft.conf"
+
+install -v -m 0644 files/home/pi/update_all.sh "${ROOTFS_DIR}/home/pi/update_all.sh"
```
Comment (builderjer) [inline stage-picroft/00-build/01-run.sh:15]: I think the default user should be ovos and not pi

File: stage-picroft/00-build/files/etc/systemd/system/getty@tty1.service.d/autologin.conf [hunk 2]
```diff
@@ -0,0 +1,3 @@
+[Service]
+ExecStart=
+ExecStart=-/sbin/agetty --autologin pi --noclear %I     38400 linux
```
Comment (builderjer) [inline stage-picroft/00-build/files/etc/systemd/system/getty@tty1.service.d/autologin.conf:3]: Same here.  User ovos instead of pi

**Review by builderjer — COMMENTED** [review]
I like this, but I think that the user should be ovos instead of pi.  And I would like to add a modified version of [cli-login.sh](https://github.com/OpenVoiceOS/ovos-buildroot/blob/develop/buildroot-external/rootfs-overlay/home/mycroft/cli_login.sh) also

**Comment by JarbasAl** [comment]
see https://downloads.openvoiceos.com/images/picroft/github/workspace/pi-gen/deploy/

**Comment by j1nx** [comment]
I really, REALLY like this work.

As part of my lists to be made, I still have the checklists for testing and checklist for feature completeness between images. Have to take care of that, however will also include this image within those list.

**Pull request merged**